packages/jest-cli/src/cli/index.js

```diff
@@ -246,9 +247,22 @@ const getConfigs = (
   }
 
   if (projects.length > 1) {
-    const parsedConfigs = projects.map(root =>
-      readConfig(argv, root, true, configPath),
-    );
+    const parsedConfigs = projects
+      .filter(root => {
+        // Ignore globbed files that cannot be `require`d.
+        if (
+          fs.existsSync(root) &&
+          !fs.lstatSync(root).isDirectory() &&
```

Comment:
should we just do a single `lstatSync` and `try-catch` it to avoid 2 separate IO operations?

Reply:
That's also a possibility.  My take is:

- `--projects` is often single to double-digits of entries, whereas the difference between 1 lookup vs. 2 wasn't discernible in my testing.  (I originally didn't have the directory check there)

- If there are any edge-cases that aren't accounted for, we'll have the opportunity to see & correct the error in the future, vs. it being silently ignored (& not running any tests for that directory).